week-3/A-array-find/exercise.js

```diff
@@ -20,7 +20,10 @@ var names = [
   "Ahmed"
 ];
 
-var longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+function findLongNameThatStartsWithA(name) {
```

Comment:
Since this function just checks a single value, it would be better named `isLongNameThatStartsWithA`. It doesn't do any finding itself. It could equally well be used with a `filter` call for example, so the context in which it is used shouldn't affect its name.